README.md

```diff
@@ -194,12 +194,7 @@ public class MyTracerConfiguration {
 
     @Bean
     public io.opentracing.Tracer jaegerTracer() {
```

Comment:
The method name should be changed it is not jaeger tracer anymore.

Reply:
Sure, that would definitely make things more consistent. @gerloh can you do that please?

Reply:
https://github.com/opentracing-contrib/java-spring-jaeger/pull/83